flambe/runner/utils.py

```diff
@@ -26,11 +26,14 @@ def get_folder_size_MB(path: str) -> float:
 
     """
     accum = 0
-    for dirpath, dirnames, filenames in os.walk(path):
-        for f in filenames:
-            fp = os.path.join(dirpath, f)
-            if os.path.exists(fp) and not os.path.islink(fp):
-                accum += os.path.getsize(fp)
+    if os.path.isdir(path):
+        for dirpath, dirnames, filenames in os.walk(path):
+            for f in filenames:
+                fp = os.path.join(dirpath, f)
+                if os.path.exists(fp) and not os.path.islink(fp):
+                    accum += os.path.getsize(fp)
```

Comment:
Do we really need to do a summation here? It shouldn't be necessary to traverse the path.

Reply:
There is no standard way of getting the size of a directory (something similar to `du`). `os.walk` seems the way to go here, even the `os.walk` docs has an example on how to do this (https://docs.python.org/3/library/os.html#os.walk)